buildstockbatch/gcp/gcp.py

```diff
@@ -641,6 +657,15 @@ def start_batch_job(self, batch_info):
         instances = batch_v1.AllocationPolicy.InstancePolicyOrTemplate(policy=policy)
         allocation_policy = batch_v1.AllocationPolicy(
             instances=[instances],
+            network=batch_v1.AllocationPolicy.NetworkPolicy(
+                network_interfaces=[
+                    batch_v1.AllocationPolicy.NetworkInterface(
+                        network="global/networks/default",
+                        subnetwork=f"regions/{self.region}/subnetworks/default",
+                        no_external_ip_address=True,
+                    )
+                ]
+            ),
```

Comment:
This looks great! Thanks, Natalie! Just a question here: once we eliminate the use of external IPs, I assume we can not simply SSH into machines for any reason like what you did earlier to confirm the remaining docker images? Are there alternatives? I don't think this would be a big issue but still trying to understand if this would impose any limitations.

Reply:
ok, great! for some reason I though we are SSHing to external IPs but this makes sense!